ingestion-beam/src/main/java/com/mozilla/telemetry/options/OutputFileFormat.java

```diff
@@ -17,6 +17,9 @@
 
     /** Return this PubsubMessage payload as text. */
     public String encodeSingleMessage(PubsubMessage message) {
+      if (message == null || message.getPayload() == null) {
+        return "";
+      }
```

Comment:
This is a behavioral change, but seems like a reasonable choice for the scenario?

Reply:
fine for whole message null, but payload null should be able to encode, because it can exist and can have attributes, such as an error message.

Reply:
Agreed that it's possible to have a message with a null payload. This particular bit of code is for `text` output where we are throwing away attributes and just encoding the payload as a String. If we don't special-case the null payload here, the next line calls `new String(null)` which throws NPE.